tensorflow/examples/learn/resnet.py

```diff
@@ -60,7 +60,7 @@ def res_net_model(features, labels, mode):
         filters=64,
         kernel_size=7,
         activation=tf.nn.relu)
-    net = tf.layers.batch_normalization(net)
+    net = tf.layers.batch_normalization(net, training=mode == tf.estimator.ModeKeys.TRAIN)
```

Comment:
Just calculate a boolean 

```training = mode == tf.estimator.ModeKeys.TRAIN```

and pass that to batch_normalization. Same everywhere.